Makefile

```diff
@@ -770,20 +770,26 @@ install.docker-full: install.docker install.docker-docs
 
 .PHONY: install.systemd
 ifneq (,$(findstring systemd,$(BUILDTAGS)))
+
+define install-systemd-unit
+	install ${SELINUXOPT} -m 644 $(1) $(2)
+	$(shell awk -i inplace '{gsub(/@@PODMAN@@/,"$(DESTDIR)$(BINDIR)/podman")}1' $(2))
```

Comment:
Thinking this through... I don't think you want `$(DESTDIR)`. Isn't that used by rpm to install into a temporary work directory? It would be Very Bad(TM) to have `/root/tmp/rpmspec/mytmpdir/usr/bin/podman` in these systemd files.

This is becoming a huge can of worms. Why exactly is this PR so important?

Reply:
$(DESTIR)$(BINDIR) is where Podman is installed to (also docs etc), so I think it's the right location.

> This is becoming a huge can of worms. Why exactly is this PR so important?

It was requested in https://github.com/containers/podman/issues/10787.  Given the time I already spent on it, I share the thought.

Reply:
Another case where `$(DESTDIR)` may break it is...for the `podman*release` targets (tarball, zip, msi, etc).  We "install" into a temp directory.

Idea: Instead of trying to fix this for every possible corner-case, maybe just print a warning about it and let the user fix it as/if needed :grinning:

Reply:
If not...consider if you can use the built-in `$(file...)` [and other native functions](https://www.gnu.org/software/make/manual/html_node/Functions.html) instead of calling out to a shell.  Might make things slightly less fragile - I've been bitten by this several times on Mac, where it's BSD sed/awk/grep/bash are *just* different enough to cause headaches.

Reply:
Oh! This might also help...systemd has [a bunch of paths pre-defined in environment variables](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Paths)